### PR TITLE
Iocgns_DatabaseIO: use `CGNS_ENUM` to refer to types

### PR DESCRIPTION
**Pull request by mathstuf**

### CODE REVIEW

**Comment by mathstuf** [comment]
Hmm. Why is the CLA bot confused on *this* PR?

**Comment by gsjaardema** [comment]
The CLA Assistant was just added.  Sandia requires external contributors to Sandia open-source software projects to indicate that they have read and signed the CLA.  Let me know if it doesn't work.

**Comment by gsjaardema** [comment]
I guess it isn't finding the file though...

**Comment by gsjaardema** [comment]
The CLA document should be reachable at https://github.com/gsjaardema/seacas/blob/master/SEACAS-CLA.md

**Comment by gsjaardema** [comment]
I think the correct change here is to delete line 1986.  It looks like maybe my merging of your previous PR resulted in a duplicate line here (old and new) instead of just the new line with the CGNS macros...

File: packages/seacas/libraries/ioss/src/cgns/Iocgns_DatabaseIO.C [hunk 1]
```diff
@@ -1983,7 +1983,7 @@ namespace Iocgns {
       const Ioss::GroupingEntity *sb   = nb->contained_in();
       int                         zone = Iocgns::Utils::get_db_zone(sb);
       int                         solution_index =
-          Utils::find_solution_index(get_file_pointer(), base, zone, step, CG_Vertex);
```
Comment (gsjaardema) [inline packages/seacas/libraries/ioss/src/cgns/Iocgns_DatabaseIO.C:1986]: This line should be deleted.  It looks like I accidentally added a duplicate line in my previous PR which was trying to add in your PR changes...

**Review by gsjaardema — CHANGES_REQUESTED** [review]

**Comment by mathstuf** [comment]
I'll have to send the CLA through our contracts team. Hopefully it won't be too long.

**Comment by gsjaardema** [comment]
Sorry 'bout that.  I will close this PR and make the change in another place just so we can get this fixed.

**Comment by mathstuf** [comment]
Closing in preference for #260.

**Pull request closed without merging**